simulation_ws/src/cloudwatch_simulation/CMakeLists.txt

```diff
@@ -3,17 +3,17 @@ project(cloudwatch_simulation)
 
 find_package(catkin REQUIRED COMPONENTS
   gazebo_ros
-  turtlebot3_description    # required to install .rviz model
+  turtlebot3_navigation  # required for copy of .rviz file
 )
 
 catkin_package()
 
-install(DIRECTORY launch worlds
+install(DIRECTORY launch worlds maps
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
 
-# Copy the rviz model for easier access in AWS RoboMaker RViz 
-install(FILES ${turtlebot3_description_DIR}/../rviz/model.rviz
+# Copy the rviz model for easier access in RoboMaker RViz 
+install(FILES ${turtlebot3_navigation_DIR}/../rviz/turtlebot3_navigation.rviz
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/rviz
-  RENAME turtlebot3_model.rviz
+  RENAME turtlebot3_navigation.rviz
```

Comment:
FWIW, `turtlebot3_navigation.rviz` is *not* the same as `turtlebot3_model.rviz`.  They configure rviz differently.  I don't think it's a big deal -- we want easier/better access from rviz tool to configs on disk/in the bundle sooner than later, which would make this `install()` work-around less important.